core/templates/dev/head/components/score-ring/score-ring.directive.html

```diff
@@ -25,7 +25,7 @@
     font-style: normal;
     font-weight: normal;
     font-size: 48px;
-    line-height: 60px;
+    line-height: 3.750px;
```

Comment:
This change seems a bit weird to me, did you mean em?

Reply:
Hi @vinitamurthi, revert this change seemed un-necessary Thanks!